ci/ray_ci/utils.py

```diff
@@ -57,6 +57,17 @@ def docker_login(docker_ecr: str) -> None:
         )
 
 
+def docker_pull(image: str) -> None:
+    """
+    Pull docker image
+    """
+    subprocess.run(
```

Comment:
I think I asked you to change this to `check_run` somewhere? do not know where that comment is now..

why is it changed back? maybe it is on the last one?

Reply:
it was on https://github.com/ray-project/ray/pull/38924 that this PR is based on; you still see it here since this hasn't been merged with master yet, it is merged now \o/